R/structural.properties.R

```diff
@@ -1540,12 +1540,7 @@ reciprocity <- reciprocity_impl
 #' edge_density(g, loops = TRUE) # this is right!!!
 #' edge_density(simplify(g), loops = FALSE) # this is also right, but different
 #'
-edge_density <- function(graph, loops = FALSE) {
-  ensure_igraph(graph)
-
-  on.exit(.Call(R_igraph_finalizer))
-  .Call(R_igraph_density, graph, as.logical(loops))
-}
+edge_density <- density_impl
```

Comment:
Same.

Reply:
Not sure I understand: do you want the following?

```R
edge_density <- function (graph, loops=FALSE) {
    density_impl(graph, loops)
} 
```

Reply:
I think so but with the ...?

Reply:
OK. Do we want to do it with a wrapper or auto-generation?